internal/consts/consts.go

```diff
@@ -4,24 +4,34 @@ const (
 	/*
 		common field names
 	*/
-	FieldPath           = "path"
-	FieldParameters     = "parameters"
-	FieldMethod         = "method"
-	FieldNamespace      = "namespace"
-	FieldNamespaceID    = "namespace_id"
-	FieldBackend        = "backend"
-	FieldPathFQ         = "path_fq"
-	FieldData           = "data"
-	FieldMount          = "mount"
-	FieldName           = "name"
-	FieldVersion        = "version"
-	FieldMetadata       = "metadata"
-	FieldNames          = "names"
-	FieldLeaseID        = "lease_id"
-	FieldLeaseDuration  = "lease_duration"
-	FieldLeaseRenewable = "lease_renewable"
-	FieldDepth          = "depth"
-	FieldDataJSON       = "data_json"
+	FieldPath                     = "path"
+	FieldParameters               = "parameters"
+	FieldMethod                   = "method"
+	FieldNamespace                = "namespace"
+	FieldNamespaceID              = "namespace_id"
+	FieldBackend                  = "backend"
+	FieldPathFQ                   = "path_fq"
+	FieldData                     = "data"
+	FieldMount                    = "mount"
+	FieldName                     = "name"
+	FieldVersion                  = "version"
+	FieldMetadata                 = "metadata"
+	FieldNames                    = "names"
+	FieldLeaseID                  = "lease_id"
+	FieldLeaseDuration            = "lease_duration"
+	FieldLeaseRenewable           = "lease_renewable"
+	FieldDepth                    = "depth"
+	FieldDataJSON                 = "data_json"
+	FieldRole                     = "role"
+	FieldDescription              = "description"
+	FieldTTL                      = "ttl"
+	FieldDefaultLeaseTTL          = "default_lease_ttl_seconds"
```

Comment:
I created this constant to refer to the field in [resource_mount.go#L38](https://github.com/hashicorp/terraform-provider-vault/blob/main/vault/resource_mount.go#L38) which is defined with the `_seconds` suffix. I don't think I can remove the suffix, since I'm using the same mount schema directly for the new k8s secrets resource. 

I could just remove this constant completely. I added it here because it seemed like something other secrets engine resources could use (if they follow a similar pattern using the mount schema).